utility/actor.go

```diff
@@ -177,11 +173,9 @@ func (u *UtilityContext) GetMaxPausedBlocks(actorType typesUtil.UtilActorType) (
 
 func (u *UtilityContext) GetMinimumPauseBlocks(actorType typesUtil.UtilActorType) (minPauseBlocks int, err typesUtil.Error) {
 	var paramName string
```

Comment:
Declaring this right above the switch statement would make it easier to follow & understand